torch/onnx/_internal/fx/exporter.py

```diff
@@ -121,6 +123,11 @@ def export_after_normalizing_args_and_kwargs(
         use_binary_format: whether to return the ONNX model in binary format.
             If False, `onnx.ModelProto` will be returned. If False, the byte array
             generated by `onnx.ModelProto.SerializeToString` is returned.
+        op_level_debug: Whether to export the model with op level debug information
+            with onnxruntime evaluator.
+        dynamic_axes: Whether to export the model with dynamic axes. This would set
+            the shape of input and nodes all to dynamic by following symbolic fx graph.
```

Comment:
Terms needs to be as explicit as possible. "following symbolic fx graph" is not a well known term.
```suggestion
            the shape of input and nodes all to dynamic by calling `proxy_tensor.make_fx` with `tracing_mode="symbolic"`.
```

Reply:
Done